bookshelf/src/app.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Create a new Silex Application with Twig.  Configure it for debugging.
+ * Follows Silex Skeleton pattern.
+ */
+use Silex\Application;
+use Silex\Provider\TwigServiceProvider;
+
+$app = new Application();
+
+$app->register(new TwigServiceProvider(), array(
+    'twig.path' => __DIR__.'/../templates',
+    'twig.options' => array(
+        'strict_variables' => false,
```

Comment:
Leaving it false for now.  Templates contain variables that are not filled now, but will be filled in future steps.